pages/index.tsx

```diff
@@ -53,9 +53,15 @@ export default function Home() {
     const currentChainAllowed = allowedChains.some(
       (allowedChain) => allowedChain.id === chain?.id
     );
-    if (isConnected && !currentChainAllowed) {
-      const defaultChainId = chains[0]?.id;
-      switchNetwork?.(defaultChainId);
+
+    const isSequenceWallet = connector?.id === "sequence";
+    const shouldSwitchToDefault =
+      isSequenceWallet && chain?.id !== defaultChainId();
```

Comment:
Personally think `shouldSwitchToDefault` should also have the `|| !currentChainAllowed`

Reply:
@har777 as Rad is OOO this week, do you mind making this addition to the PR? We'll then merge afterwards :-)

Reply:
updated